### PR TITLE
Varun

### PR DESCRIPTION
**Pull request by Techvarun1991**
I have done some basic changes in some of the components, will you please do a code review for that if changes are there please let me know....

### CODE REVIEW

File: src/Components/Calender.js
Comment (YJB28) [inline src/Components/Calender.js:1]: Done with that..!!

File: src/Components/Dialog.js [hunk 1]
```diff
@@ -32,6 +32,10 @@ export default function FormDialog({ prop, style, buttonTitle, icon ,variant,ic}
       
       <Dialog open={open} onClose={handleClose} maxWidth={style} fullWidth>
         <DialogContent>{prop}</DialogContent>
+
+
+
+        
         {/* <DialogActions>
           <Button onClick={handleClose}>Cancel</Button>
```
Comment (YJB28) [inline src/Components/Dialog.js:40]: Remove Unwanted Comments and Spaces

File: src/Components/GraphVisualization.js [hunk 2]
```diff
@@ -77,6 +77,8 @@ const GraphVisualization = (props) => {
       // })
```
Comment (YJB28) [inline src/Components/GraphVisualization.js:77]: Remove commented part

File: src/Components/cool-background1.svg
Comment (YJB28) [inline src/Components/cool-background1.svg:1]: Why you Deleted that component?

**Review by YJB28 — CHANGES_REQUESTED** [review]
Done the Commented Changes

File: src/Components/Calender.js
Comment (YJB28) [reply src/Components/Calender.js:1]: Ok Thanks

Comment (Techvarun1991) [reply src/Components/Calender.js:1]: Ok Thanks!!

File: src/Components/Dialog.js (hunk 1 above)
Comment (Techvarun1991) [reply src/Components/Dialog.js]: Removed As Per Instructions

File: src/Components/cool-background1.svg
Comment (Techvarun1991) [reply src/Components/cool-background1.svg:1]: Actually it was a svg file it was taken only for testing purpose

File: src/Components/GraphVisualization.js (hunk 2 above)
Comment (Techvarun1991) [reply src/Components/GraphVisualization.js]: Some commented part is there for the labels on the links(lines) it is for future use so i have kept it as it is

Comment (YJB28) [reply src/Components/GraphVisualization.js]: ok then Keep at it is.

**Review by YJB28 — APPROVED** [review]
All resolved.

**Pull request merged**